### DISCUSSION TITLE
Issues with sensor setup in airgap

### DISCUSSION THREAD

**Discussion by constantlyinstalling** [Security-Onion-Solutions/securityonion#7666] [Unsupported Versions] [answered]
I have been running into an issue when deploying forward nodes and search nodes in an airgapped environment. The manager will be set up and functional and then when I attempt to run the setup on a forward/search node I will be prompted for how I want to connect to the internet rather than being asked for the manager's ip address. When prompted for the internet my setup will fail because my distributed cluster is airgapped. Is there a solution to this issue? I run into it about half the time I deploy and have to either reinstall and pray it doesn't happen again or beat my head against the keyboard until it finally doesn't prompt me to connect to the internet.

**Comment by dougburks** [comment, marked as answer]
Are you using the latest version of our Security Onion ISO image?

Are you sure your manager is set to airgap?  When a node joins to the master, it should inherit that airgap setting:
https://github.com/Security-Onion-Solutions/securityonion/issues/5914

**Reply by constantlyinstalling** [reply]
I am sure that my manager is set to airgap, I'm currently using 2.3.100 but I've also seen this issue on the latest ISO (though not as often)

**Reply by constantlyinstalling** [reply]
I just tried leaving it with default DNS and it worked this time, I'm not sure why it wasn't working with my DNS server.
